### DISCUSSION TITLE
Filter list does not seem to work

### DISCUSSION THREAD

**Discussion by NN-Binary** [open-webui/open-webui#4343] [General]
Hi!

I'm trying to restrict to myself via docker flags the list of available models:

`docker run -d -p 8080:8080 -e ENABLE_MODEL_FILTER=true -e MODEL_FILTER_LIST="openai/gpt-4o;meta-llama/llama-3.1-405b-instruct;openai/gpt-4o-mini;anthropic/claude-3.5-sonnet"	-e ENABLE_OAUTH_SIGNUP=False -e ENABLE_OLLAMA_API=True -e ENABLE_ADMIN_EXPORT=False -e ENABLE_COMMUNITY_SHARING=False -e WEBUI_BUILD_HASH="AI" -e DEFAULT_USER_ROLE=user -e ENABLE_SIGNUP=False -e WEBUI_NAME="AI" -e ENABLE_LOGIN_FORM=False -e SHOW_ADMIN_DETAILS=False -e WEBUI_AUTH=False -e OPENAI_API_KEY=XXXXX -e CUSTOM_NAME="AI" -e OPENAI_API_BASE_URL=https://openrouter.ai/api/v1 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

I've tried many different filter list to follow what's in the documentation but I can't seem to reach a fixed list.

Thank you!

**Comment by TheMasterFX** [comment]
The Whitelist/Model filter only applies to Users, not admins: https://docs.openwebui.com/tutorial/whitelist/

**Comment by Sunnnner** [comment]
<img width="448" alt="image" src="https://github.com/user-attachments/assets/816c0c6b-deba-4d7f-a2b7-d17e95a3f523">
Again I set the environment variable here and found that it didn't override my previous manual settings

![image](https://github.com/user-attachments/assets/0dbc797f-642e-4917-9eec-d32610537727)

**Comment by Sunnnner** [comment]
## prescription:
Open the webui.db database, open the config table, modify the configuration file data, delete the model_filter content, save the config table, and then use the docker environment variable to set the
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/1dd237c5-3484-4705-87a8-3c108b36741c">

<img width="815" alt="image" src="https://github.com/user-attachments/assets/ab2e6e8f-9cba-4c91-a49c-97bfb001df68">
